scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+start = datetime(2019, 5, 15)
+test_bugs = bugzilla.download_bugs_between(start, datetime.now())
+
+m = DuplicateModel()
+m.train()
```

Comment:
You are downloading the bugs to evaluate, and then training the model on all bugs. This means the model will be trained on the bugs to evaluate too.